content/Overview/architecture.mdx

```diff
@@ -0,0 +1,16 @@
+---
+slug: architecture
+title: Architecture
+---
+
+To better understand the architecture and workflow with Configu, it can be helpful to familiarize yourself with the various components that make up the platform. Here is a summary of the main components of Configu:
+
+- `Configu CLI` is a lightweight binary that is available for every major operating system, Docker, and CI/CD tools. It provides a user-friendly interface for managing configurations through the command line.
+- `Configu SDK` is a library that can be configured as a dependency in your application. It allows you to manage your configurations as code and provides a simple and consistent way to access configuration data in your application.
+- `ConfigSchema file (<uid>.cfgu.json)` is a human-readable, open-sourced format for declaring configurations and their characteristics. It is designed to be stored in your source control alongside the code that uses these configurations, and is safe to be treated as code.
+- `Configu SaaS Configuration Store` is a web interface tool that provides a one-stop-shop for all configuration needs, from development to production. It offers advanced capabilities such as automation, visualization, testing, security, and tracking.
+- `Configu IDE plugin` is a plugin that integrates stright to the development environment (IDE) and provides immediate feedback on configuration issues.
+
+The below diagram demostrates an high level flow using Configu's OSS components toghther with Configu SaaS Configuration Store.
+
+![Configu's SaaS Architecture model](./architecture-diagramm.jpg)
```

Comment:
Please update to the new diagram, this one have a VS code instead of version control